### PR TITLE
feat(clerk-js): Add glass background to cards

### PR DESCRIPTION
**Pull request by desiprisg**
## Description

Since with v5 components we want to use the glass effect for the navbars/footers, it is necessary that these colors are a product of the background color. We can leverage `linear-gradient` to stack two colors on top of each other, and use an opacity on the background color in order for the blur to be visible, thus creating the glass effect. 

Default theme:

Screenshot 2024-01-15 at 20 21 34

Dark theme:
Screenshot 2024-01-15 at 20 22 04

No need to add anything to `dark.ts` or add something to the appearance prop. This should work out of the box.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by octoper** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @octoper, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-ikc4e6af7.clerkstage.dev) | Jan 15, 2024 06:37 PM |

File: packages/clerk-js/src/ui/elements/Navbar.tsx [hunk 1]
```diff
@@ -295,7 +299,11 @@ export const NavbarMenuButtonRow = ({ navbarTitleLocalizationKey, ...props }: Na
       elementDescriptor={descriptors.navbarMobileMenuRow}
       sx={t => ({
         display: 'none',
-        background: `linear-gradient(${t.colors.$blackAlpha100},${t.colors.$blackAlpha100}), linear-gradient(${t.colors.$colorBackground}, ${t.colors.$colorBackground})`,
+        background: common.mergedColorsBackground(
+          colors.setAlpha(t.colors.$colorBackground, 0.8),
+          t.colors.$blackAlpha50,
+        ),
+        backdropFilter: 'blur(10px)',
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/elements/Navbar.tsx:306]: Shall we add a `t.backdropFilter.$defaultBlur` ?

**Review by panteliselef — APPROVED** [review]
💯  Looks good, seems like the preview works for safari and firefox. Just a minor comment about the duplication of `blur()`

**Review by nikosdouvlis — APPROVED** [review]
Nice!

**Pull request merged**